app/views/publications/mid_cycle_report/show.html.erb

```diff
@@ -0,0 +1,141 @@
+<% if @national_data.nil? %>
+  <p class="govuk-body">This report is not ready to view.</p>
+<% else %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">
+        National recruitment performance for the 2022 to 2023 recruitment cycle
+      </h1>
+      <p class="govuk-body">These statistics show the recruitment performance of all initial teacher training (ITT) providers in England from 11 October 2022 to 3 April 2023.</p>
+      <p class="govuk-body">If your organisation is an ITT provider, and you use the Manage teacher training applications (Manage) service, you can view your own organisation's recruitment performance by signing in to <%= govuk_link_to('Manage', provider_interface_sign_in_path) %>.</p>
+      <h2 class="govuk-heading-m">How the data in this report compares to other data we publish</h2>
+      <p class="govuk-body">The page showing <%= govuk_link_to('monthly statistics for courses starting in the 2023 to 2024 academic year', publications_monthly_report_itt_path(RecruitmentCycle.current_year)) %>, calculates data in the same way UCAS used to.</p>
+      <p class="govuk-body">This page about national recruitment performance, calculates data in a new, improved way. For this reason, you should not compare it with the monthly statistics page. </p>
+      <h2 class="govuk-heading-m">Candidates who've submitted applications</h2>
+      <p class="govuk-body">This table shows the number of candidates who’ve submitted at least one application to an ITT provider from 10 October 2022 to 3 April 2023.</p>
+    </div>
+  </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <table class="govuk-table govuk-!-margin-bottom-9">
```

Comment:
Move the `<table>` and `</table>` to the partial maybe? Feels weird to separate the table header, body and container from its footer? Then rename the partial `_table`. 😄